app/client/views/confirmation/confirmation.html

```diff
@@ -257,8 +257,26 @@
             </div>
           </div>
 
+          <div class="field" style="padding-top: 20px">
+            <p>
+              VTHacks will offer travel reimbursements to students who carpool from schools that are far away from Blacksburg, Virginia.
+              If you would like to apply for a travel reimbursement, please mark the box below and fill out the necessary info.
+            </p>
+            <p>
+              Preference will be given to participants who are able to carpool with other hackers.
+            </p>
+          </div>
+
+          <div class="field">
+            <label> Are you interested in travel reimbursement? </label>
+            <div class="ui checkbox">
+              <input type="checkbox" ng-model="user.confirmation.travel.userWantsReimbursement">
+              <label> Yeah I like driving with people, why not </label>
```

Comment:
Can you add a `for="user.confirmation.travel.userWantsReimbursement"` option for this label tag, and also one for `user.confirmation.userNeedsTransportation` above?